draft-ietf-quic-http.md

```diff
@@ -732,9 +732,10 @@ initial GOAWAY frame with the last Stream ID set to the maximum possible value
 for a client-initiated, bidirectional stream (i.e. 2^62-4 in case of QUIC
 version 1).  This GOAWAY frame signals to the client that shutdown is imminent
 and that initiating further requests is prohibited.  After allowing time for any
-in-flight requests (at least one round-trip time), the server would send another
-GOAWAY frame with an updated last Stream ID.  This ensures that a connection can
-be cleanly shut down without causing requests to fail.
+in-flight requests to reach the server, the server MAY send another GOAWAY frame
+indicating which requests it will accept before the end of the connection. This
```

Comment:
Now that we do not use RFC 2119 keywords in describing the procedure of gracefully shutting down the connection, it might be a good idea to change:
"the server MAY send another GOAWAY frame indicating which requests it will accept before the end of the connection."
to: "the server will send another GOAWAY frame indicating the requests it accepted."

Reply:
Don't you mean "can send"?

Reply:
My view has been "server sends" is better than "can send", because this sentence is part of what a server "can" do to gracefully shutdown a connection. But I'll leave that up to the editors.

Reply:
Perhaps, but "can" is used other places through this description.  Unless we rewrite the whole thing to be can-free, I'm choosing consistency.